### ISSUE TITLE
Response.entities is not immediately available after `await this.api().get(...)`

### ISSUE THREAD

**Issue by alex-kowalczyk** [vuex-orm/plugin-axios#89]
The following code logs that `response.entities` is null, even though later (e.g. when clicking 'expand' triangle' in chrome's console log) the data is there - see the screenshot. I haven't found anything in documentation on that behavior. Is it a bug, or should something more be awaited?
```
static async getData() {
  const response = await this.api().get(uri);
  console.log('after': response);
}
```
![image](https://user-images.githubusercontent.com/7422175/69007864-ddfc9c00-0943-11ea-8faf-7fc3ffc20afd.png)

**Comment by alex-kowalczyk** [comment]
https://github.com/vuex-orm/plugin-axios/blob/ba4843401eb7570007074266e4c5362930baf0c9/src/api/Request.ts#L151

I believe this line misses `await` - should be:
``config.save && await response.save()``

**Comment by alex-kowalczyk** [comment]
Possibly it's caller (async Request.request)  shoul await this.createResponse too:
https://github.com/vuex-orm/plugin-axios/blob/ba4843401eb7570007074266e4c5362930baf0c9/src/api/Request.ts#L122
`return await this.createResponse(axiosResponse, requestConfig)`

**Comment by kiaking** [comment]
Oh, thank you for the report! Yeah, you sounds pretty much right about it. We should be adding it.

**Comment by kiaking** [comment]
This is now fixed in Vuex ORM 0.9.1! 🎉 
https://github.com/vuex-orm/plugin-axios/releases/tag/v0.9.1
